packages/core/content-manager/shared/contracts/history-versions.ts

```diff
@@ -12,7 +12,8 @@ export interface CreateHistoryVersion {
   locale: string | null;
   status: 'draft' | 'published' | 'modified' | null;
   data: Record<string, unknown>;
-  schema: Struct.SchemaAttributes;
+  schema: Schema.Attributes;
```

Comment:
Did you understand the difference between this and Struct.SchemaAttributes? I thought this one required a generic UID argument

Reply:
you're right, didn't mean to overwrite that